dts/msm8916/msm8939-qrd-wt82918.dts

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+        qcom,msm-id = <239 0>;
+        qcom,board-id = <0x1000b 16>;
+
+        model = "Lenovo Vibe K5 Plus (Wingtech WT82918)";
+        compatible = "wingtech,wt82918", "qcom,msm8939", "lk2nd,device";
+
+        panel {
+                compatible = "wingtech,wt82918-panel";
+
+                qcom,mdss_dsi_otm1901a_tm_1080p_video {
+                        compatible = "wingtech,otm1901a-tm";
+                };
+		qcom,mdss_dsi_boe_nt35595_1080p_video {
```

Comment:
```suggestion
                };

		qcom,mdss_dsi_boe_nt35595_1080p_video {
```

Please keep a newline between nodes. Also please note that the dts files are intented with 8-wide hard tab, not spaces.

Reply:
Done